content/en/docs/setup/production-environment/tools/kubespray.md

```diff
@@ -56,7 +56,7 @@ After you provision your servers, create an [inventory file for Ansible](https:/
 
 Kubespray provides the ability to customize many aspects of the deployment:
 
-* Choice deployment mode: kubeadm or non-kubeadm
+* Choice deployment mode: kubeadm or non-kubeadm(deprecated non-kubeadm deployment starting from v2.8)
```

Comment:
Should be:
* Choice deployment mode: kubeadm or non-kubeadm (deprecated since v2.8)

Optionally: v2.8 is more than two years ago. Maybe we should remove the non-kubeadm mode. For example:
* Deployment mode: kubeadm

Reply:
I think we should remove this list item. Let's not describe two options as a choice when one is deprecated.

Reply:
Any news on this, @wangxf1987? :)

Reply:
non-kubeadm deployment has been removed on Kubespray side since https://github.com/kubernetes-sigs/kubespray/commit/ddffdb63bfcc65a1731a16d316ce10d4903e3261 which is 2.5 years ago.

So +1 for removing this list item to avoid confusion.

Reply:
The above description about non-kubeadm has been removed from the document on Kubespray side with https://github.com/kubernetes-sigs/kubespray/pull/7834

Reply:
@wangxf1987 could you update as the above?